libs/blocks/icon-block/icon-block.css

```diff
@@ -168,6 +178,10 @@
   margin-bottom: var(--spacing-s);
 }
 
+html:not([dir='rtl']).icon-block .foreground .text-content p[class*="body-m"]:last-of-type {
```

Comment:
This would target `html` tags that have an `icon-block` class name, I'm not sure this is the desired behaviour here.

Reply:
Block has been removed after reviewing and re-evaluating the code.